### PR TITLE
DEV: Limit reaction_users to 27

### PR DESCRIPTION
**Pull request by Ahmedgagan**
@jjaffeux Did 27 here because
- when I drop a reaction, in the reaction-state-panel if there are users more than say 53, it should display `26 and 27 others...` but the issue here is I dropped a reaction which means 1 user is removed from 26 and it displays 25 users instead of 26.

### CODE REVIEW

File: assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6 [hunk 1]
```diff
@@ -317,6 +317,11 @@ export default createWidget("discourse-reactions-actions", {
           return false;
         } else if (reaction.id === post.current_user_reaction.id) {
           post.reactions[index].count -= 1;
+          const users = post.reactions[index].users;
+          users.splice(
+            users.indexOf(users.findBy("username", "ahmedgagan6")),
```
Comment (jjaffeux) [inline assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6]: ? 😅

File: assets/javascripts/discourse/widgets/discourse-reactions-state-panel-reaction.js.es6 [hunk 2]
```diff
@@ -17,6 +17,8 @@ export default createWidget("discourse-reactions-state-panel-reaction", {
 
   html(attrs) {
     const elements = [];
+    const MAX_USERS_COUNT = 26;
```
Comment (jjaffeux) [inline assets/javascripts/discourse/widgets/discourse-reactions-state-panel-reaction.js.es6]: make it top level

File: plugin.rb [hunk 3]
```diff
@@ -62,12 +62,13 @@ class Engine < ::Rails::Engine
   end
 
   add_to_serializer(:post, :reactions) do
+    MAX_USERS_COUNT = 26
```
Comment (jjaffeux) [inline plugin.rb]: make it top level

File: assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6 (hunk 1 above)
Comment (Ahmedgagan) [reply assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6]: when a user drops a reaction, we were not removing the reaction-user, so removed it here

Comment (jjaffeux) [reply assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6]: read the code again please :p

Comment (jjaffeux) [reply assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6]: You hardcoded your username: ahmedgagan6

Comment (Ahmedgagan) [reply assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6]: oooh got it😂  a real mess😜

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**